packages/react-magma-dom/src/components/Select/Select.test.js

```diff
@@ -500,6 +504,124 @@ describe('Select', () => {
     expect(getByTestId('customClearIndicator')).toBeInTheDocument();
   });
 
+  describe('additional content', () => {
+    const helpLinkLabel = 'Learn more';
+
+    const onHelpLinkClick = () => {
+      alert('Help link clicked!');
+    };
+
+    it('Should accept additional content to the right of the select label', () => {
+      const { getByTestId } = render(
+        <Select
+          additionalContent={
+            <Tooltip content={helpLinkLabel}>
+              <IconButton
+                aria-label={helpLinkLabel}
+                icon={<HelpIcon />}
+                onClick={onHelpLinkClick}
+                testId={'Icon Button'}
+                type={ButtonType.button}
+                size={ButtonSize.small}
+                variant={ButtonVariant.link}
+              />
+            </Tooltip>
+          }
+          labelText={labelText}
+          items={items}
+        />
+      );
+
+      expect(getByTestId('Icon Button')).toBeInTheDocument();
+    });
+
+    it('Should accept additional content to the right of the multi-select label', () => {
+      const { getByTestId } = render(
+        <Select
+          additionalContent={
+            <Tooltip content={helpLinkLabel}>
+              <IconButton
+                aria-label={helpLinkLabel}
+                icon={<HelpIcon />}
+                onClick={onHelpLinkClick}
+                testId={'Icon Button'}
+                type={ButtonType.button}
+                size={ButtonSize.small}
+                variant={ButtonVariant.link}
+              />
+            </Tooltip>
+          }
+          labelText={labelText}
+          isMulti
+          items={items}
+        />
+      );
+
+      expect(getByTestId('Icon Button')).toBeInTheDocument();
+    });
+
+    it('Should accept additional content to display inline with the label and select', () => {
+      const { getByTestId } = render(
+        <Select
+          additionalContent={
+            <Tooltip content={helpLinkLabel}>
+              <IconButton
+                aria-label={helpLinkLabel}
+                icon={<HelpIcon />}
+                onClick={onHelpLinkClick}
+                type={ButtonType.button}
+                size={ButtonSize.small}
+                variant={ButtonVariant.link}
+              />
+            </Tooltip>
+          }
+          labelPosition="left"
+          labelText={labelText}
+          items={items}
+          data-testid="selectContainerElement"
+        />
+      );
+
+      expect(getByTestId('selectContainerElement')).toBeInTheDocument();
+      expect(getByTestId('selectContainerElement')).toHaveStyleRule(
+        'display',
+        'flex'
+      );
+    });
+
+    it('Should accept additional content to display along select with a visually hidden label', () => {
```

Comment:
"When label position is left and isLabelVisuallyHidden is true, should accept ...."

Reply:
I think it's really helpful for the test description to be very specific about the props being tested, then mention the result.